### DISCUSSION TITLE
Update to Features page to include MariaDB

### DISCUSSION THREAD

**Discussion by robertsilen** [sqlalchemy/sqlalchemy#11344] [Ideas]
Hello, [sqlalchemy.org/features.html ](https://www.sqlalchemy.org/features.html) page mentions "MySQL" as supported. It would make sense to update it to "MySQL and MariaDB" corresponding to the [docs page with the same title](https://docs.sqlalchemy.org/en/20/dialects/mysql.html). How can I suggest this to somebody with access to the home page? Cheers, Robert

**Comment by CaselIT** [comment]
Hi,

That page is indeed out of date, since come of those dialects are no longer supported internally

**Reply by CaselIT** [reply]
created https://github.com/sqlalchemyorg/sqlalchemyorg/pull/19. Please follow up there.

If you have edit suggestions on the test feel free to comment there!
